### PR TITLE
Just Try

### PR DESCRIPTION
**Pull request by BenFielmann**
-i have simply added new strokes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
+<p>
+    Die Ursprünge der Tasse reichen bis zur Zeit der Tang-Dynastie (618–906 n. Chr.) in China zurück,
+    als das Porzellan erfunden wurde. Ihren Weg nach Europa fand die damalige 
+    Tasse Überlieferungen zufolge durch portugiesische Händler, welche etwa 1610 n. Chr. 
+    den ersten Tee aus China nach England importierten.
+</p>
+<!-------------------------->
+<section>
+    ><ol id="list">
+        <h3>Arten</h3>
```
Comment (mbantwerpes) [inline index.html:38]: I think h3 tags should not belong into lists. If you want the first element to be bigger/bold it should probably be applied to the first element of the list with css.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Tasse</title>
+    <link href="css" rel="stylesheet">
+    <scribt src="js" defer></scribt>
```
Comment (christophschmitz) [inline index.html:9]:
```suggestion
    <scribt src="js" defer></scribt>
```
<script src="js" defer></script>

**Review by christophschmitz — APPROVED** [review]
Interesting topic - looking good. I think <script> is the right tag.

File: index.html (hunk 1 above)
Comment (BenFielmann) [reply index.html:38]:
```suggestion
        <h3>Arten</h3>
        absolutely right thanks for the feedback, will be implemented.
```

**Pull request merged**